### DISCUSSION TITLE
Main email on account deleted

### DISCUSSION THREAD

**Discussion by icboureanu** [community/community#71081] [New to GitHub]
The main  email registered to a github account has been deleted. How can I get into that account?

**Comment by queenofcorgis** [comment]
If you are still able to login to that account, you can just update and verify a new primary email, [here are the steps](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/changing-your-primary-email-address). 

If you're unable to get into the account, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
